website/docs/getting-started/features.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Features
+sidebar_position: 2
+---
+
+# Features provided by odo
+
+By using odo, application developers can develop, test, debug and deploy microservices based applications on Kubernetes without having a deep understanding of the platform.
+
+odo follows "create and push" workflow for almost everything. It means, as a user, when you "create" something the information (or manifest) is stored in a configuration file, and then upon doing a "push" it gets created  on the Kubernetes cluster. One can take an existing git repository and create an odo component from it, which can be pushed to a Kubernetes cluster.
+
+odo helps "deploy and link" multiple components and services with each other. Using odo, developers can create and deploy services based on [Kubernetes Operators](https://github.com/operator-framework/) in their [development cluster](cluster-setup/development-clusters). These services can be created using any of the Operators available on [OperatorHub.io](https://operatorhub.io). Next, upon linking this service, odo injects the service configuration into the microservice created using odo. Your application can use this configuration to communicate with the Operator backed service. 
+
+
+### What can odo do?
+
+odo uses container images to run the microservices in the cluster.
+
+Full details of what each odo command is capable of doing can be found in the "Command Reference" sections.
```

Comment:
Perhaps add a link to "Command Reference" like you did "development cluster" in the earlier section?

Reply:
"Command Reference" is a top level section. Can't have a link to it. It'll throw a 404. The "development cluster" is a separate doc so it could be linked.

Reply:
Oh right, okay then.